pypeit/par/pypeitpar.py

```diff
@@ -1763,9 +1767,22 @@ def __init__(self, obj_toler=None, assign_obj=None, nsig_thrshd=None,
         defaults['extract_missing_objs'] = False
         dtypes['extract_missing_objs'] = bool
         descr['extract_missing_objs'] = 'Force extraction of undetected objects at the location expected ' \
-                                        'from the slitmask design. PypeIt will try to determine the FWHM from ' \
-                                        'the flux profile (by using ``find_fwhm`` in `FindObjPar` as initial guess). ' \
-                                        'If the FWHM cannot be determined, ``find_fwhm`` will be assumed.'
+                                        'from the slitmask design.'
```

Comment:
These params should be moved to find_objects or extraction parsers, but not here.